viz/app/components/ErrorBoundary.tsx

```diff
@@ -28,65 +26,14 @@ export class ErrorBoundary extends React.Component<
 
   componentDidCatch(error: unknown) {
     this.setState({ hasError: true, error });
+    this.props.onErrored();
```

Comment:
Why not passing the error message as we use to?

Reply:
We don't do anything with it yet

Reply:
So we'll pass it when we need it